geopandas/_compat.py

```diff
@@ -90,3 +91,38 @@ def set_use_pygeos(val=None):
 
 
 set_use_pygeos()
+
+
+def import_optional_dependency(name: str, extra: str = ""):
+    """
+    Import an optional dependency.
+
+    Adapted from pandas.compat._optional::import_optional_dependency
+
+    Raises a formatted ImportError if the module is not present.
+
+    Parameters
+    ----------
+    name : str
+        The module name.
+    extra : str
+        Additional text to include in the ImportError message.
+    Returns
+    -------
+    module
+    """
+    msg = (
+        f"Missing optional dependency '{name}'. {extra} "
```

Comment:
f-strings should only be from python 3.6, and we are still supporting 3.5 (we should discuss dropping that, but that's for another issue :))